### ISSUE TITLE
Error message when the addon tries to setup interface options

### ISSUE THREAD

**Issue by m-thorsen** [jbolduan/Click-To-Cast-Tooltip#7]
Game version: 11.0.0.55933 (pre-expansion patch)
Addon version: 1.4.0

```
Message: ...k-To-Cast-Tooltip/Click-To-Cast-Tooltip-Settings.lua:90: attempt to call global 'InterfaceOptions_AddCategory' (a nil value)
Time: Thu Aug  1 00:32:00 2024
Count: 1
Stack: ...k-To-Cast-Tooltip/Click-To-Cast-Tooltip-Settings.lua:90: attempt to call global 'InterfaceOptions_AddCategory' (a nil value)
[string "@Interface/AddOns/Click-To-Cast-Tooltip/Click-To-Cast-Tooltip-Settings.lua"]:90: in function `InitializeOptions'
[string "@Interface/AddOns/Click-To-Cast-Tooltip/Click-To-Cast-Tooltip-Settings.lua"]:39: in function <...k-To-Cast-Tooltip/Click-To-Cast-Tooltip-Settings.lua:23>
```

**Comment by jbolduan** [comment]
I was noticing this today as well; it appears to be affecting some other addons as well.  I'll poke around and find a fix!  Thank you for the report!

**Comment by jbolduan** [comment]
This should be fixed in 1.4.1.  Which will be showing up shortly.  Let me know if you continue to have issue with the settings.

**Comment by jbolduan** [comment]
Marking closed.
